web/src/popover_menus.js

```diff
@@ -46,6 +48,11 @@ let message_actions_popover_keyboard_toggle = false;
 let compose_control_buttons_popover_instance;
 let starred_messages_popover_instance;
 
+let drafts_popover_instance;
+export function is_drafts_visible() {
```

Comment:
Let's call this `is_drafts_menu_visible`; there's a separate drafts overlay we don't want to confuse this is.

Reply:
Thank you for the suggestion. I completely agree with you. 

But, after the refactoring, such function no longer exists ;) I hope it is for better code organization.